design/census-import/config.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+require('dotenv').config()
+const os = require('os')
+const toBool = require('to-bool')
+
+const twoMinutesInMilliseconds = 120000
+const oneMinuteInMilliseconds = 60000
+
+const getEnvironment = () => {
+  return process.env.ENVIRONMENT_NAME || 'Local-Dev'
+}
+
+module.exports = {
+  Logging: {
+    LogLevel: process.env.LOG_LEVEL || 'info',
+    LogDna: {
+      key: process.env.LOGDNA_API_KEY,
+      hostname: `${os.hostname()}:${process.pid}`,
+      ip: undefined,
+      mac: undefined,
+      app: `MTCAdmin:${getEnvironment()}`,
+      env: `${getEnvironment()}`
+    },
+    Express: {
+      UseWinston: process.env.EXPRESS_LOGGING_WINSTON || false
+    },
+    SendToAppInsights: process.env.APPINSIGHTS_WINSTON_LOGGER || false
+  },
+  Monitoring: {
```

Comment:
This config is surprising - it should only have values in it that are required by the import.  A lot of this stuff seems unnecessary.

Reply:
@jon-shipley I have extracted the sql config parts in `sql.config.js` and kept only the logging and monitoring parts in `config.js`